### PR TITLE
Adjust context menu and use open colors

### LINKED ISSUE #531: Keep labels in one line for the context menu
Screenshot 2020-01-23 at 9 00 30 PM

### PR DESCRIPTION
**Pull request by lipis**
Fixes #531

### CODE REVIEW

File: src/components/ContextMenu.css [hunk 1]
```diff
@@ -7,28 +7,24 @@
   user-select: none;
   margin: -0.25rem 0 0 0.125rem;
   padding: 0.25rem 0;
-  background-color: #f2f2f2;
-  border: 1px solid #bcbcbc;
-}
-
-.context-menu__option {
```
Comment (dwelle) [inline src/components/ContextMenu.css]: This actually results in two UI regressions:

1. it's too bunched together:
     
     image

2. the position calculation doesn't work when there's no min width:

      image

So to ensure the items don't wrap, you'll prob need to calculate the min width during open, before calculating the position.

**Comment by vjeux** [comment]
I used MacOS colors. I think that they are good and don't think it's worth using a different palette.

File: src/components/ContextMenu.css (hunk 1 above)
Comment (lipis) [reply src/components/ContextMenu.css]: I'll check those.. Thanks for testing

Comment (lipis) [reply src/components/ContextMenu.css]: Fixed the width issue.. check it again.

**Comment by lipis** [comment]
Fixed the word wrap thing and should be better now.. I used the open colors for consistency across the app.

File: src/components/ContextMenu.css (hunk 1 above)
Comment (dwelle) [reply src/components/ContextMenu.css]: Yep, working. That's actually a simple and elegant solution.

Comment (dwelle) [reply src/components/ContextMenu.css]: WTBS, I'd still add `padding` to emulate the previous style.

**Comment by dwelle** [comment]
Reverted colors & padding to gain consensus. Merging. Thanks...

**Pull request merged**